test/data/import/old/1.5.1.json

```diff
@@ -16,6 +16,7 @@
                     "documentation": "",
                     "method": "get",
                     "endpoint": "answer",
+                    "randomResponse": false,
```

Comment:
Please do not modify files in `test/data/migrations`, `test/data/import/new` and `test/data/import/old`